sonar-erlang-plugin/src/main/java/org/sonar/plugins/erlang/ErlangSquidSensor.java

```diff
@@ -52,30 +51,21 @@
 
 import org.sonar.squidbridge.api.SourceFile;
 
-import org.sonar.squidbridge.api.SourceFunction;
-
-import org.sonar.squidbridge.indexer.QueryByParent;
-
 import org.sonar.squidbridge.indexer.QueryByType;
 import org.sonar.sslr.parser.LexerlessGrammar;
 
 import java.io.Serializable;
 import java.util.ArrayList;
 import java.util.Collection;
 import java.util.List;
+import java.util.Objects;
 
 public class ErlangSquidSensor implements Sensor {
 
   private static final Logger LOG = Loggers.get(ErlangSquidSensor.class);
-
```

Comment:
Is this functionality completely removed from SonarQube, e.g., shouldn't we just comment it out, if it must be reworked?

Reply:
It is completely deprecated, here is an example: http://javadocs.sonarsource.org/latest/apidocs/org/sonar/api/measures/CoreMetrics.html#FUNCTION_COMPLEXITY_DISTRIBUTION_KEY